modules/serverless-ossm-traffic-isolation-architecture.adoc

```diff
@@ -0,0 +1,43 @@
+:_content-type: CONCEPT
+[id="serverless-ossm-traffic-isolation-architecture_context"]
+= High-level architecture
+
+The high-level architecture of {ServerlessProductShortName} traffic isolation provided by {SMProductShortName} consists of `AuthorizationPolicies` in the `knative-serving`, `knative-eventing` and the tenants namespaces, with all the components being part of the {SMProductShortName}. The injected {SMProductShortName} sidecars take care of enforcing those rules to isolate network traffic between tenants.
```

Comment:
Rewrite to reduce wordiness: "The injected {SMProductShortName} sidecars enforce those rules to isolate network traffic between tenants."